src/entities/transactions/transaction.ts

```diff
@@ -0,0 +1,34 @@
+import { ProtocolType } from '../protocolTypes';
+import { TransactionStatus } from './transactionStatus';
+import { TransactionType } from './transactionType';
+
+export interface RainbowTransaction {
+  address?: string;
```

Comment:
what's the `address` of a transaction (I'm asking because we have from & to)

Reply:
This address is for the asset address (we also bring up the symbol and name)